core/ArchiveProcessor.php

```diff
@@ -327,7 +327,18 @@ public function insertNumericRecord($name, $value)
      */
     public function insertBlobRecord($name, $values)
     {
-        $this->archiveWriter->insertBlobRecord($name, $values);
+        $newInsert = array();
+
+        if (!empty($values[0])) {
```

Comment:
I think it's not expected for `empty($values[0]) == true`, right? Would a log statement be potentially helpful in this case?